mjolnir/input/read_global_potential.hpp

```diff
@@ -502,16 +502,21 @@ read_uniform_lennard_jones_potential(const toml::value& global)
     //     # ...
     // ]
 
-    if( ! global.contains("parameters"))
+    const auto& ps = toml::find_or<toml::array>(global, "parameters", {});
+    if(ps.size() == 0)
     {
+        MJOLNIR_LOG_WARN("UniformLennardJonesPotential does not have any participants.");
+        MJOLNIR_LOG_WARN("All the particles are considered to be participants.");
+        MJOLNIR_LOG_WARN("To disable this potential, "
+                             "simply remove the part from the input file.");
```

Comment:
```suggestion
                             "simply remove or comment the part out from the input file.");
```
I now think it is better to mention about comment.